src/op_handlers/context.rs

```diff
@@ -76,7 +76,7 @@ pub fn set_context_u128(vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmEr
     Ok(())
 }
 
-pub fn aux_mutating0(_vm: &mut VMState, _opcode: &Opcode) -> Result<(), EraVmError> {
+pub fn _aux_mutating0(_vm: &mut VMState, _opcode: &Opcode) -> Result<(), EraVmError> {
```

Comment:
Seeing the new implementation of `unimplemented` this no longer seems to be necessary

Reply:
I left it for when we have to implement it